Makefile

```diff
@@ -279,6 +272,16 @@ bundle-register:
 	&& cd ${DIR} \
 	&& ${ROOTPATH}/devops/scripts/register_bundle_with_api.sh --acr-name "$${ACR_NAME}" --bundle-type "$${BUNDLE_TYPE}" --current --insecure --tre_url "$${TRE_URL}" --verify --workspace-service-name "$${WORKSPACE_SERVICE_NAME}"
 
+shared-service-register-and-deploy:
+	@# NOTE: ACR_NAME below comes from the env files, so needs the double '$$'. Others are set on command execution and don't
+	$(call target_title, "Registering ${DIR} bundle") \
```

Comment:
Title is wrong

Reply:
With everything else I forgot to fix it, but I'll add a note to do it in my follow-up PR